clients/client-accessanalyzer/README.md

```diff
@@ -3,4 +3,203 @@
 [![NPM version](https://img.shields.io/npm/v/@aws-sdk/client-accessanalyzer/latest.svg)](https://www.npmjs.com/package/@aws-sdk/client-accessanalyzer)
 [![NPM downloads](https://img.shields.io/npm/dm/@aws-sdk/client-accessanalyzer.svg)](https://www.npmjs.com/package/@aws-sdk/client-accessanalyzer)
 
-For SDK usage, please step to [SDK readme](https://github.com/aws/aws-sdk-js-v3).
+## Description
+
+AWS SDK for JavaScript AccessAnalyzer Client for Node.js, Browser and React Native.
```

Comment:
Should we add and asterisk here stating that whether the clients runs on browser depends on the CORS setting on serverside?

Reply:
Created a follow-up task to be taken after `v3.3.0` release https://github.com/aws/aws-sdk-js-v3/issues/1913